poolinroot6.err6 line 4: pool6 declared outside of network
